lib/stove-testing-e2e-wiremock/src/main/kotlin/com/trendyol/stove/testing/e2e/wiremock/WireMockSystem.kt

```diff
@@ -218,6 +218,84 @@ class WireMockSystem(
     return this
   }
 
+  @WiremockDsl
+  fun mockPostRequestContaining(
+    url: String,
+    requestContaining: Map<String, Any>,
+    statusCode: Int = 200,
+    responseBody: Option<Any> = None,
+    urlPatternFn: (url: String) -> UrlPattern = { urlEqualTo(it) }
+  ): WireMockSystem = mockPostConfigure(url, urlPatternFn) { builder, serde ->
+
+    requestContaining.forEach { (key, value) ->
+      builder.withRequestBody(
+        matchingJsonPath(
+          "$.$key",
```

Comment:
I think this works with only first level of JSON matching. If we were to make a search inside a deep Json path it wouldn’t work, would it? A test that proves it works no matter what the depth is, and a test that proves conditions for matching works as “and” operator would be good, what do you think?